Vagrantfile

```diff
@@ -47,6 +50,26 @@ Vagrant.configure(2) do |config|
     do
         echo unset $LC  >> /home/vagrant/.bashrc
     done
+
+    # Install VirtualBox guest additions
+    cd /tmp
+    VBOX_VERSION=$1
+    VBOX_ADDITIONS_URL="http://download.virtualbox.org/virtualbox/VER/VBoxGuestAdditions_VER.iso"
+    sudo wget -q -c ${VBOX_ADDITIONS_URL//VER/$VBOX_VERSION} -O VBoxGuestAdditions_$VBOX_VERSION.iso
```

Comment:
This fails in Ubuntu 14.04, 

    --2016-09-22 14:43:22--  http://download.virtualbox.org/virtualbox/4.3.36_Ubuntu/VBoxGuestAdditions_4.3.36_Ubuntu.iso
    Resolving download.virtualbox.org (download.virtualbox.org)... 62.254.36.51, 62.254.36.9
    Connecting to download.virtualbox.org (download.virtualbox.org)|62.254.36.51|:80... connected.
    HTTP request sent, awaiting response... 404 Not Found
    2016-09-22 14:43:24 ERROR 404: Not Found.

I think it would be a lot simpler to install the package with the Guest additions:

    apt-get install -y virtualbox-guest-additions-iso

We can then mount the iso from `/usr/share/virtualbox/VBoxGuestAdditions.iso` and we don't need to play games with the version, guessing URLs or anything.